### DISCUSSION TITLE
خطایی پیش آمده است، لطفا دقایقی دیگر خرید خود را تکمیل کنید Response: -10

### DISCUSSION THREAD

**Discussion by samanbahrekazemi** [Sina-Soltani/Parbad#263] [Q&A]
سلام وقت بخیر این خطا برای ZarinPal مشاهده میشه

**Comment by samanbahrekazemi** [comment]
خطایی پیش آمده است، لطفا دقایقی دیگر خرید خود را تکمیل کنید Response: -10

**Reply by Sina-Soltani** [reply]
درود

فکر نمیکنم به این پکیج ارتباطی داشته باشه. لطفا با خود درگاه در میون بزارید ببینید مشکل چیه
اگر مشخص شد از درگاه نیست، لطفا اطلاع بدید
موفق باشید

**Reply by samanbahrekazemi** [reply]
مشکل از خود درگاه بود - خطا مربوط به Ip سرور میشد که البته تغییری ندادیم تیکت زدیم به بانک و حل شد مشکل ممنون از راهنماییتون.
